src/components/stats/tx-steps.tsx

```diff
@@ -1,54 +1,57 @@
 'use client'
 
 import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card'
-import { Bar, BarChart, Line, LineChart, ResponsiveContainer } from 'recharts'
+import { TransactionStats } from '@/lib/types'
+import { XAxis, Line, Legend, LineChart, ResponsiveContainer } from 'recharts'
 
-const data = [
-  {
-    revenue: 10400,
-    subscription: 10240,
-  },
-  {
-    revenue: 14405,
-    subscription: 10300,
-  },
-  {
-    revenue: 9400,
-    subscription: 8200,
-  },
-  {
-    revenue: 8200,
-    subscription: 7278,
-  },
-  {
-    revenue: 7000,
-    subscription: 10189,
-  },
-  {
-    revenue: 9600,
-    subscription: 10239,
-  },
-  {
-    revenue: 11244,
-    subscription: 10278,
-  },
-  {
-    revenue: 16475,
-    subscription: 7189,
-  },
-]
-
-export function TxStepsStats({ className }: { className?: string }) {
+export function TxStepsStats({
+  className,
+  transactionStats,
+}: {
+  className?: string
+  transactionStats: TransactionStats
+}) {
+  const data = Array.from({ length: 7 }, (_, i) => i).map((i) => {
+    const date = new Date(
+      new Date().setDate(new Date().getDate() - (6 - i) - 1),
+    )
+    return {
+      name: `${date.getDate()}/${date.getMonth() + 1}`,
+      transactionsCount: transactionStats.transactionsCountLast7Days[i],
+      stepsNumber: transactionStats.stepsNumberLast7Days[i] / 100000,
```

Comment:
Why do you divide it by 100_000 here? Let's just make two different y-axes?

Reply:
I'm not sure how to proceed with this, could you provide some guidance?
It looks like the data tile is too tiny to actually display correctly 2 different Y axes and I couldn't find how to proceed in recharts docs.

Reply:
If it's not possible to handle with recharts, let's just normalize both (transactionsCount, stepsNumber) values to avoid using the constant 100_000.

Reply:
Thanks for the guidance this is what I've implemented in my latest commit.